add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
```

Comment:
Consider using prepared statements to prevent SQL injection attacks. Directly using user input in SQL queries is a security risk. PHP's `mysqli` library supports prepared statements which can mitigate this risk.

```php
$stmt = $connection->prepare("INSERT INTO books (title, author, year) VALUES (?, ?, ?)");
$stmt->bind_param("ssi", $title, $author, $year);
if ($stmt->execute()) {
    echo "Book added successfully!";
} else {
    echo "Error: " . $stmt->error;
}
$stmt->close();
```